HW17/utilities/config_reader.py

```diff
@@ -21,9 +22,9 @@ def get_browser_id():
 
 
 def get_headless_status():
-    result = my_config.get("browser_id", "headless")
-    if result in ["True", "true", "1"]:
-        return True
-    else:
-        return False
+    path_to_config_file = f"{ROOT_DIR}/configurations/config.json"
+    with open(path_to_config_file, "r") as file:
+        config = json.load(file)
+    return config["headless"].lower() == "true" and config["browser_id"] == 1
```

Comment:
why do you need this part of the code?

Reply:
i wan't to be able turn off/on headless mode - as you can see i saved it in config.json

Reply:
I understand it, by why do you need this function? 
you can pass it where use need just using env.headless